tests/ReportbackApiTest.php

```diff
@@ -83,6 +84,10 @@ public function testErrorOnDuplicateReportback()
             'remote_addr' => '207.110.19.130',
         ]);
 
-        $this->assertEquals(500, $response->status());
+        $this->assertEquals(200, $response->status());
```

Comment:
This is totally subjective, but I kinda like using Laravel's test case methods for things like this:

```php
$this->call('POST', $this->reportbackApiUrl, [ ... ]);

$this->assertResponseStatus(200);
$this->seeJsonSubset([
    'data' => [
        'quantity' => 2000,
    ],
]);
```